envoy/api/v2/bootstrap/bootstrap.proto

```diff
@@ -31,18 +32,18 @@ message Bootstrap {
   message StaticResources {
     // Static :ref:`Listeners <envoy_api_msg_Listener>`. These listeners are
     // available regardless of LDS configuration.
-    repeated Listener listeners = 1;
+    repeated listener.Listener listeners = 1;
 
     // If a network based configuration source is specified for :ref:`cds_config
     // <envoy_api_field_Bootstrap.DynamicResources.cds_config>`, it's necessary
     // to have some initial cluster definitions available to allow Envoy to know
     // how to speak to the management server. These cluster definitions may not
     // use :ref:`EDS <arch_overview_dynamic_config_sds>` (i.e. they should be static
     // IP or DNS-based).
-    repeated Cluster clusters = 2;
+    repeated cluster.Cluster clusters = 2;
 
     // [#not-implemented-hide:]
-    repeated Secret secrets = 3;
+    repeated cert.Secret secrets = 3;
```

Comment:
Do we want `cert` subpackage or a more general `security` one? @PiotrSikora

Reply:
I think if we go with `security`, we should put `auth` in there, too.